src/test/java/mate/intro/service/AttachmentServiceImplTest.java

```diff
@@ -0,0 +1,124 @@
+package mate.intro.service;
+
+import mate.intro.dto.attachment.AttachmentDto;
+import mate.intro.dto.attachment.CreateAttachmentRequestDto;
+import mate.intro.dto.dropbox.DropBoxResponseDto;
+import mate.intro.exception.EntityNotFoundException;
+import mate.intro.exception.RegistrationException;
+import mate.intro.mapper.AttachmentMapper;
+import mate.intro.model.Attachment;
+import mate.intro.model.Task;
+import mate.intro.repository.AttachmentRepository;
+import mate.intro.repository.TaskRepository;
+import mate.intro.service.impl.AttachmentServiceImpl;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import static org.mockito.ArgumentMatchers.any;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@ExtendWith(MockitoExtension.class)
+class AttachmentServiceImplTest {
+    @InjectMocks
+    AttachmentServiceImpl attachmentService;
```

Comment:
Place after Mocks

Reply:
Do you mean, I should replace fild with annotation @InjectMocks after fields with annotation @Mock in each service test?